src/main/java/org/carlmontrobotics/subsystems/Drivetrain.java

```diff
@@ -292,16 +304,22 @@ public void periodic() {
             // module.move(0, goal);
         }
 
-        field.setRobotPose(odometry.getPoseMeters());
+        // field.setRobotPose(odometry.getPoseMeters());
+
+        field.setRobotPose(poseEstimator.getEstimatedPosition());
 
+        // odometry.update(gyro.getRotation2d(), getModulePositions());
 
-        odometry.update(gyro.getRotation2d(), getModulePositions());
+        poseEstimator.update(Rotation2d.fromDegrees(getHeading()), getModulePositions());
```

Comment:
Should it use `gyro.getRotation2d()` instead of `Rotation2d.fromDegrees(getHeading())` (that is what was used for the old odometry)